community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/PhysicalWritableLogChannel.java

```diff
@@ -141,5 +163,6 @@ private ByteBuffer bufferWithGuaranteedSpace( int spaceInBytes ) throws IOExcept
     public void close() throws IOException
     {
         emptyBufferIntoChannelAndClearIt();
+        closed = true;
```

Comment:
That's true. Although it turns out that the way we grab the logFile monitor that will never happen. Change my comments thereabout?